multinet/util.py

```diff
@@ -104,3 +106,19 @@ def data_path(file_name: str) -> str:
 def generate_arango_workspace_name() -> str:
     """Generate a string that can be used as an ArangoDB workspace name."""
     return f"w-{uuid1()}"
+
+
+def get_or_init_flask_secret_key() -> str:
+    """Load or create the flask secret key."""
+    env_file_path = Path(".env")
+    dotenv.load_dotenv(env_file_path)
+
+    secret_key = dotenv.get_key(env_file_path, "FLASK_SECRET_KEY")
+
+    if not secret_key:
+        secret_key = uuid4().hex
+        ok, _, secret_key = dotenv.set_key(
+            env_file_path, "FLASK_SECRET_KEY", secret_key
+        )
```

Comment:
We should probably have a live discussion about this. I'm still not convinced this should affect the .env file on disk. But beyond that, we are definitely breaking the dotenv protocols here (I think). I don't fully understand it, but I think there's a cascade of filenames that are checked, things like .env.local, etc., beyond which, the dotenv system supports different files in development and production. We're not making use of that sort of thing now, but whenever we depart from a standard, pain usually results in the future.

Reply:
I've updated the behavior with ab3b7d7, lmk what you think

Reply:
Thanks for this change.

The function still looks like it returns multiple values on consecutive calls (if the environment variable is unset). I mentioned this elsewhere, so if I can find that thread, I will resolve this one.

Reply:
Ah found it: https://github.com/multinet-app/multinet-server/pull/358#discussion_r416002910.